src/server/app.ts

```diff
@@ -0,0 +1,72 @@
+/******************************************************************************
```

Comment:
This is worth thinking about in future, we should probably define our own application interface that accepts interfaces for dependencies like loggers, and infrastructure (i.e. backends), etc.

I.e. we should move to a dependency injection model. 

I think this can be a future improvement story. 

@vijayg10 , @PaulGregoryBaker let me know what you guys think.

Reply:
Sure, that sounds interesting. I want to see an example of the use-case if there are any available.